IntegrationTests/UI.IntegrationTests/NBugReports/BugReportFormTests.cs

```diff
@@ -9,26 +9,26 @@ namespace GitExtensions.UITests.NBugReports
     [TestFixture]
     public class BugReportFormTests
     {
-        private BugReportForm _form;
+        ////private BugReportForm _form;
```

Comment:
What shall happen with these tests?

Reply:
Once the whole thing is finished I'll see whether these tests are still relevant, and, if they are, where they belong.